src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
you should better define a `Random` field in this class, so that you don't create a new instance of it every time this method is called
```suggestion
    private final Random random = new Random();
    
    public String getRandomColor() {
```